cardano-cli/src/Cardano/CLI/EraBased/Options/Governance/Committee.hs

```diff
@@ -88,3 +89,27 @@ pGovernanceCommitteeKeyHash era = do
     pCmd w =
       GovernanceCommitteeKeyHash w
         <$> pVerificationKeyFileIn
+
+pGovernanceCommitteeCreateColdKeyResignationCertificate :: ()
+  => CardanoEra era
+  -> Maybe (Parser (GovernanceCommitteeCmds era))
+pGovernanceCommitteeCreateColdKeyResignationCertificate =
+  featureInEra
```

Comment:
why not `maybeFeatureInEra` instead?

Reply:
`maybeFeatureInEra` isn't the right function to use.  There is a theoretically a better function to use than `featureInEra`, but it hasn't been written unfortunately.